File: IHIS/IHIS.Framework/IHIS.CloudConnector/Contracts/Models/Clis/CLIS2015U02GrdProtocolInfo.cs
using System;

namespace IHIS.CloudConnector.Contracts.Models.Clis
{
    [Serializable]
    public class CLIS2015U02GrdProtocolInfo
    {
        private String _protocolId;
        private String _protocolCode;
        private String _protocolName;
        private String _fromDate;
        private String _toDate;
        private String _protocolStatus;
        private String _description;
        private String _resource;
        private String _rowState;

        public String ProtocolId
        {
            get { return this._protocolId; }
            set { this._protocolId = value; }
        }

        public String ProtocolCode
        {
            get { return this._protocolCode; }
            set { this._protocolCode = value; }
        }

        public String ProtocolName
        {
            get { return this._protocolName; }
            set { this._protocolName = value; }
        }

        public String FromDate
        {
            get { return this._fromDate; }
            set { this._fromDate = value; }
        }

        public String ToDate
        {
            get { return this._toDate; }
            set { this._toDate = value; }
        }

        public String ProtocolStatus
        {
            get { return this._protocolStatus; }
            set { this._protocolStatus = value; }
        }

        public String Description
        {
            get { return this._description; }
            set { this._description = value; }
        }

        public String Resource
        {
            get { return this._resource; }
            set { this._resource = value; }
        }

        public String RowState
        {
            get { return this._rowState; }
            set { this._rowState = value; }
        }

        public CLIS2015U02GrdProtocolInfo() { }

        public CLIS2015U02GrdProtocolInfo(String protocolId, String protocolCode, String protocolName, String fromDate, String toDate, String protocolStatus, String description, String resource, String rowState)
        {
            this._protocolId = protocolId;
            this._protocolCode = protocolCode;
            this._protocolName = protocolName;
            this._fromDate = fromDate;
            this._toDate = toDate;
            this._protocolStatus = protocolStatus;
            this._description = description;
            this._resource = resource;
            this._rowState = rowState;
        }

    }
}

File: MED-CLOUD/med-data/src/main/java/nta/med/data/model/ihis/clis/CLIS2015U02GrdProtocolInfo.java
package nta.med.data.model.ihis.clis;

import java.util.Date;

public class CLIS2015U02GrdProtocolInfo {
	private Integer protocolId;
    private String protocolCode;
    private String protocolName;
    private Date fromDate;
    private Date toDate;
    private String protocolStatus;
    private String description;
    private String resource;
	public CLIS2015U02GrdProtocolInfo(Integer protocolId, String protocolCode,
			String protocolName, Date fromDate, Date toDate,
			String protocolStatus, String description, String resource) {
		super();
		this.protocolId = protocolId;
		this.protocolCode = protocolCode;
		this.protocolName = protocolName;
		this.fromDate = fromDate;
		this.toDate = toDate;
		this.protocolStatus = protocolStatus;
		this.description = description;
		this.resource = resource;
	}
	public Integer getProtocolId() {
		return protocolId;
	}
	public void setProtocolId(Integer protocolId) {
		this.protocolId = protocolId;
	}
	public String getProtocolCode() {
		return protocolCode;
	}
	public void setProtocolCode(String protocolCode) {
		this.protocolCode = protocolCode;
	}
	public String getProtocolName() {
		return protocolName;
	}
	public void setProtocolName(String protocolName) {
		this.protocolName = protocolName;
	}
	public Date getFromDate() {
		return fromDate;
	}
	public void setFromDate(Date fromDate) {
		this.fromDate = fromDate;
	}
	public Date getToDate() {
		return toDate;
	}
	public void setToDate(Date toDate) {
		this.toDate = toDate;
	}
	public String getProtocolStatus() {
		return protocolStatus;
	}
	public void setProtocolStatus(String protocolStatus) {
		this.protocolStatus = protocolStatus;
	}
	public String getDescription() {
		return description;
	}
	public void setDescription(String description) {
		this.description = description;
	}
	public String getResource() {
		return resource;
	}
	public void setResource(String resource) {
		this.resource = resource;
	}
}
